src/module.c

```diff
@@ -1422,8 +1442,9 @@ NAPI_METHOD(dcn_send_msg) {
   NAPI_ARGV_UINT32(chat_id, 1);
 
   //TRACE("calling..");
-  dc_msg_t* dc_msg;
+  dc_msg_t* dc_msg = NULL;
   napi_get_value_external(env, argv[2], (void**)&dc_msg);
```

Comment:
This caused a crash on windows since the memory is not automatically cleared (contains garbage data).